test/performance/tests/test_util/config.py

```diff
@@ -25,6 +26,12 @@
 # SNAP_NAME is the name of the snap under test.
 SNAP_NAME = os.getenv("TEST_SNAP_NAME") or "k8s"
 
+# KUBE_BURNER_URL is the version of kube-burner to use.
+KUBE_BURNER_URL = (
+    os.getenv("TEST_KUBE_BURNER_VERSION")
+    or "https://github.com/kube-burner/kube-burner/releases/download/v1.2/kube-burner-1.2-Linux-x86_64.tar.gz"
```

Comment:
when does this ever change?  Should we keep it up-to-date?

Reply:
We could bump the version in the future or overwrite it with a url for a different architecture. 
As discussed with Hue in an earlier comment our friends at kube burner are not consistently naming their release urls so I decided to make the entire url configurable and not just `arch` and `version`.

Reply:
Dealing with this in part 3.